t/t7600-fixup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+test_description='.'
+
+export GIT_TEST_DEFAULT_INITIAL_BRANCH_NAME
+
+TEST_PASSES_SANITIZE_LEAK=true
+. ./test-lib.sh
```

Comment:
I found this https://github.com/felipec/sharness

Replacing test-lib with sharness I'm able to run this test